pilot/pkg/config/kube/crd/client.go

```diff
@@ -31,28 +31,18 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/runtime/serializer"
 	"k8s.io/apimachinery/pkg/util/wait"
-
-	"istio.io/istio/pkg/log"
 	// import GKE cluster authentication plugin
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+
```

Comment:
`istio.io` imports should be moved to line 44 after the block `k8s.io` imports

Reply:
done